src/jira/client/jira-client.ts

```diff
@@ -463,18 +466,27 @@ const truncateIssueKeys = (repositoryObj) => {
 
 interface IssueKeyObject {
 	issueKeys?: string[];
+	associations?: JiraAssociation[];
 }
 
 export const getTruncatedIssuekeys = (data: IssueKeyObject[] = []): IssueKeyObject[] =>
 	data.reduce((acc: IssueKeyObject[], value: IssueKeyObject) => {
 		// Filter out anything that doesn't have issue keys or are not over the limit
-		if (value.issueKeys && value.issueKeys.length > ISSUE_KEY_API_LIMIT) {
+		if (value?.issueKeys && value.issueKeys.length > ISSUE_KEY_API_LIMIT) {
```

Comment:
Actually since the FF is removed, will it means that there's no `issueKeys` at all? All we have are just `associations`?
Or are you just concerning about existing msg in the queue?

Reply:
there are issueKeys for other jira entities, not just deployments.  Not all of them uses associations yet.